src/mahoji/commands/farming.ts

```diff
@@ -198,6 +216,25 @@ export const farmingCommand: OSBMahojiCommand = {
 			});
 			return `You will now use ${tier.item.name} by default.`;
 		}
+		if (options.auto_farm_filter) {
+			const autoFarmFilterString = Object.keys(autoFarm_filter_enum).find(i =>
+				stringMatches(i, options.auto_farm_filter!.auto_farm_filter_data)
+			);
+			if (!autoFarmFilterString) return 'Invalid auto farm filter.';
+			const autoFarmFilter = autoFarmFilterString as autoFarm_filter_enum;
+
+			await mahojiUserSettingsUpdate(userID, {
+				minion_autoFarmFilterToUse: autoFarmFilter
+			});
+
+			const filterTexts = {
+				allfarm: 'All crops will be farmed with the highest available seed',
+				replant: 'Only planted crops will be replanted, using the same seed'
+			};
+			type FilterKey = keyof typeof filterTexts;
```

Comment:
put these elsewhere, or atleast at the top scope. make the filterTexts `as const`, and type it properly: `Record<EnumName, string>`